src/Resources/views/Profiler/pheanstalk.html.twig

```diff
@@ -1,13 +1,14 @@
 {% extends 'WebProfilerBundle:Profiler:layout.html.twig' %}
 
 {% block toolbar %}
-    <div class="sf-toolbar-block">
-        <div class="sf-toolbar-icon">
-            <a href="{{ path('_profiler', { 'token': token, 'panel': name }) }}">
-                <span style="width:0px; height: 28px; vertical-align: middle;"></span>
-                <span class="sf-toolbar-status">{{ collector.jobCount }}</span> Jobs
-            </a>
-        </div>
+    <div class="sf-toolbar-block sf-toolbar-block-leezy-pheanstalk sf-toolbar-status-normal">
+        <a href="{{ path('_profiler', { 'token': token, 'panel': name }) }}">
+            <div class="sf-toolbar-icon">
+                {{ include('@LeezyPheanstalk/Profiler/database.svg') }}
+                <span class="sf-toolbar-value">{{ collector.jobCount }}</span>
+                <span class="sf-toolbar-label">Jobs</span>
+            </div>
+        </a>
         <div class="sf-toolbar-info">
             <div class="sf-toolbar-info-piece">
                 <b>Pheanstalks</b>
```

Comment:
@armetiz  no problem.

To facilitate @resurtm , I had in my code the following:
```twig
            <div class="sf-toolbar-info-piece">
                <b>Pheanstalks</b>
                <span class="sf-toolbar-status">{{ collector.pheanstalks|length }}</span>
            </div>
            <div class="sf-toolbar-info-piece">
                <b>Tubes</b>
                <span class="sf-toolbar-status">{{ collector.tubes|length }}</span>
            </div>
            <div class="sf-toolbar-info-piece">
                <b>Jobs</b>
                <span class="sf-toolbar-status">{{ collector.jobCount }}</span>
            </div>
```

Which gives:
![spectacle n11382](https://cloud.githubusercontent.com/assets/5219975/21849772/c2778ce4-d807-11e6-926d-e2f4a5d91ce4.png)

Oh, right. I made that SVG icon for "queues", if you like it is MIT licensed (derived from font-awesome), so free merge.
```svg
<svg width="24" height="24" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path fill="#aaaaaa" d="m 129.76988,309.16276 c -10.27512,0 -18.7154,3.30272 -25.32084,9.90817 -6.605426,6.60542 -9.908148,15.04573 -9.908148,25.32081 l 0,70.45796 c 0,10.27509 3.302722,18.71539 9.908148,25.32081 6.60544,6.60546 15.04572,9.90817 25.32084,9.90817 l 1532.46032,0 c 10.2751,0 18.7154,-3.30271 25.3208,-9.90817 6.6055,-6.60542 9.9081,-15.04572 9.9081,-25.32081 l 0,-70.45796 c 0,-10.27508 -3.3026,-18.71539 -9.9081,-25.32081 -6.6054,-6.60545 -15.0457,-9.90817 -25.3208,-9.90817 z m 140.9159,342.01901 c -29.35749,0 -54.31134,10.27509 -74.86158,30.82533 -20.55025,20.55025 -30.82535,45.5041 -30.82535,74.86158 l 0,211.37386 c 0,29.35749 10.2751,54.31136 30.82535,74.86156 20.55024,20.5502 45.50409,30.8254 74.86158,30.8254 l 211.37385,0 c 29.35748,0 54.31134,-10.2752 74.86156,-30.8254 20.55025,-20.5502 30.82536,-45.50407 30.82536,-74.86156 l 0,-211.37386 c 0,-29.35748 -10.27511,-54.31133 -30.82536,-74.86158 -20.55022,-20.55024 -45.50408,-30.82533 -74.86156,-30.82533 z m 519.62739,0 c -29.35749,0 -54.31134,10.27509 -74.86158,30.82533 -20.55025,20.55025 -30.82537,45.5041 -30.82537,74.86158 l 0,211.37386 c 0,29.35749 10.27512,54.31136 30.82537,74.86156 20.55024,20.5502 45.50409,30.8254 74.86158,30.8254 l 211.37373,0 c 29.3576,0 54.3113,-10.2752 74.8616,-30.8254 20.5502,-20.5502 30.8254,-45.50407 30.8254,-74.86156 l 0,-211.37386 c 0,-29.35748 -10.2752,-54.31133 -30.8254,-74.86158 -20.5503,-20.55024 -45.504,-30.82533 -74.8616,-30.82533 z m 519.62733,0 c -29.3576,0 -54.3114,10.27509 -74.8617,30.82533 -20.5502,20.55025 -30.8253,45.5041 -30.8253,74.86158 l 0,211.37386 c 0,29.35749 10.2751,54.31136 30.8253,74.86156 20.5503,20.5502 45.5041,30.8254 74.8617,30.8254 l 211.3739,0 c 29.3574,0 54.3112,-10.2752 74.8615,-30.8254 20.5503,-20.5502 30.8253,-45.50407 30.8253,-74.86156 l 0,-211.37386 c 0,-29.35748 -10.275,-54.31133 -30.8253,-74.86158 -20.5503,-20.55024 -45.5041,-30.82533 -74.8615,-30.82533 z M 129.76988,1277.5696 c -10.27512,0 -18.7154,3.3028 -25.32084,9.9082 -6.605426,6.6055 -9.908148,15.0457 -9.908148,25.3208 l 0,70.4579 c 0,10.2752 3.302722,18.7154 9.908148,25.3208 6.60544,6.6056 15.04572,9.9083 25.32084,9.9083 l 1532.46032,0 c 10.2751,0 18.7154,-3.3027 25.3208,-9.9083 6.6055,-6.6054 9.9081,-15.0456 9.9081,-25.3208 l 0,-70.4579 c 0,-10.2751 -3.3026,-18.7153 -9.9081,-25.3208 -6.6054,-6.6054 -15.0457,-9.9082 -25.3208,-9.9082 z" /></svg>
```
```